.pre-commit-config.yaml

```diff
@@ -28,7 +28,7 @@ repos:
         exclude: |
           (?x)(
             ^integration-tests/sceptre-project/config/|
-            ^integration-tests/sceptre-project/templates/jinja/|
+            ^integration-tests/sceptre-project/templates/|
```

Comment:
this has already been committed to master.  could you please update this PR with master?